SQL/0000-00-00-schema.sql

```diff
@@ -2355,3 +2355,14 @@ CREATE TABLE `issues_watching` (
   KEY `fk_issues_watching_2` (`issueID`),
   CONSTRAINT `fk_issues_watching_1` FOREIGN KEY (`userID`) REFERENCES `users` (`UserID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+DROP TABLE IF EXISTS `monitoring`;
+CREATE TABLE `monitoring` (
+  `CandID` int(6) NOT NULL,
+  `visit_label` varchar(255) DEFAULT NULL,
```

Comment:
It's not like there was any consistency in Loris, but I definitely support choosing `snake_case` (or `camelCase` or `PascalCase`) and sticking with it instead of promoting `WTF_case`